daras_ai_v2/bot_integration_widgets.py

```diff
@@ -391,14 +393,65 @@ def web_widget_config(bi: BotIntegration, user: AppUser | None):
             if display_pic:
                 bi.photo_url = display_pic
         else:
-            if gui.button(f"{icons.camera} Change Photo"):
-                gui.session_state["--update-display-picture"] = True
-                gui.rerun()
-        bi.name = gui.text_input("###### Name", value=bi.name)
-        bi.descripton = gui.text_area(
-            "###### Description",
-            value=bi.descripton,
+            gencol1, gencol2 = gui.columns(2)
+            with gencol1:
+                if gui.button(f"{icons.camera} Change Photo"):
+                    gui.session_state["--update-display-picture"] = True
+                    gui.rerun()
+            with gencol2:
+                generate_prompt = """Your job is to define the schema for a chatbot, using the title, description and prompt of an AI Workflow.
```

Comment:
isn't this prompt supposed to be in a /llm example?

Reply:
You're right, I removed the generate_prompt var